### ISSUE TITLE
How can i get the arena icon in-game ?

### ISSUE THREAD

**Issue by riftflopper** [Milxnor/Project-Reboot-3.0#102]
Can anybody tell me how I could get the arena point indicator icon to work in-game? Please I don't want any point system or any kind of stuff like that I just want the point indicator icon (look at the picture below if you don't know what I mean)

![image](https://github.com/user-attachments/assets/6585dfbd-f52c-44f7-9dfc-9722d94e2932)

**Comment by Milxnor** [comment]
This is backend related only on earlier versions then it becomes a gameserver thing which I don't know how to do

**Comment by riftflopper** [comment]
Oh alright man do you know some backend devs that have chapter 2 season 2+ arena that i could maybe ask ?

**Comment by Milxnor** [comment]
Not sure sorry,I don't know if anyone has figured out proper arena icon on S10+. Most servers just use a pak to add the widget to the playlist which I suppose is fine.

**Comment by riftflopper** [comment]
It's fine my bro. Last question do you know how to make one or get one from somebody ? (I'm so sorry if I'm annoying you by asking so much about this).

**Comment by fraudian** [comment]
most projects wont just "give" the pak out, you usually have to make it your self.

**Comment by Milxnor** [comment]
wym bro cant u just go in paks folder and take it?

**Comment by fraudian** [comment]
i dont know lol sorry lmao xd ahaha

**Comment by riftflopper** [comment]
> most projects wont just "give" the pak out, you usually have to make it your self.

yeah i know i probably have to use idk like fngameproj or something to make one

**Comment by riftflopper** [comment]
> wym bro cant u just go in paks folder and take it?

if that would work it would be lit but no ones doing 14.40 and i heard its the hardest version to mod
